var/spack/repos/builtin/packages/netcdf-cxx4/package.py

```diff
@@ -56,6 +67,15 @@ def libs(self):
     def configure_args(self):
         config_args = []
 
+        # We need to build with MPI wrappers if either of the parallel I/O
+        # features is enabled in netcdf-c:
+        # https://www.unidata.ucar.edu/software/netcdf/docs/building_netcdf_fortran.html
+        netcdf_c_spec = self.spec['netcdf-c']
+        if '+mpi' in netcdf_c_spec or '+parallel-netcdf' in netcdf_c_spec:
+            config_args.append('CC=%s' % self.spec['mpi'].mpicc)
+            config_args.append('FC=%s' % self.spec['mpi'].mpifc)
+            config_args.append('F77=%s' % self.spec['mpi'].mpif77)
```

Comment:
I doubt that setting `FC` and `F77` is needed for this package. I would add setting `CXX` and check whether `CC` is needed. In any case, thanks for trying to make this consistent.

Reply:
Of course, good point. I'll remove Fortran and check if we still need to set CC.